src/ClassDiscovery.php

```diff
@@ -120,9 +121,9 @@ public static function setStrategies(array $strategies)
     /**
      * Append a strategy at the end of the strategy queue.
      *
-     * @param DiscoveryStrategy $strategy
+     * @param string $strategy Fully qualified class name to a DiscoveryStrategy
      */
-    public static function appendStrategy(DiscoveryStrategy $strategy)
+    public static function appendStrategy($strategy)
```

Comment:
Please correct me if Im wrong. But the problem here is that `appendStrategy($strategy)` takes a FQCN not an object. I could add another `DiscoveryHelper` but it feels weird.. doesn't it?